src/array_api_extra/_lib/_funcs.py

```diff
@@ -172,7 +172,7 @@ def create_diagonal(
     Parameters
     ----------
     x : array
-        A 1-D array.
+        An array having shape (*broadcast_dims, k).
```

Comment:
Nit, but at least in SciPy we'd call this `batch_dims`. I'll let @lucascolley decide what language he'd prefer

Reply:
Sure, let me know

Reply:
Either way possibly should be monospace ?
```suggestion
        An array having shape ``(*broadcast_dims, k)``.
```

Reply:
No strong opinion either way, so let's go for `batch_dims` to match SciPy

Reply:
Already done!  :smile: